test/test_config.hh.in

```diff
@@ -10,11 +10,7 @@
     gz::rendering::TestValues())
 
 /// \todo(anyone) re-enable ogre2 test once ogre 2.2 works on macOS
-#ifdef __APPLE__
-static const std::vector<const char *> kRenderEngineTestValues{"ogre", "optix"};
-#else
-static const std::vector<const char *> kRenderEngineTestValues{"ogre2", "optix"};
-#endif
+static const std::vector<const char *> kRenderEngineTestValues{"ogre2"};
```

Comment:
if we are going to disable tests for optix, I would prefer doing that in a separate pull request rather than burying it here

Reply:
restored